generators/server/templates/pom.xml.ejs

```diff
@@ -1155,7 +1155,29 @@
         </profile>
         <%_ if (!skipClient) { _%>
         <profile>
-            <id>webpack</id>
+            <id>dev-webpack</id>
```

Comment:
to be consistent letc call it `webpack-dev`

Reply:
btw how is this triggered in default dev profile?

Reply:
originally you had to pass -Pdev,webpack to trigger this, ideally it should just work with -Pdev and with default `./mvnw`

Reply:
Renamed to webpack-dev

If the `target/www` does't exist, it will use the new `webpack-dev` maven profile, using `dev` spring boot profile (so dev properties).
If it exists, it will use the default `dev` maven profile, using `dev` spring boot profile.

Tested with:
- mvn clean, mvn, stop and start mvn
- mvn clean, mvn -Pdev, stop and start mvn -Pdev